README.md

```diff
@@ -1,2 +1,107 @@
 # go-cruise-control
-It's client library written in Golang for interacting with Linkedin Cruise Control using its HTTP API.
+
+It's client library (written in Golang) for interacting with
+[Linkedin Cruise Control](https://github.com/linkedin/cruise-control) using its HTTP API.
+
+## How to use it
+
+```shell
+go get github.com/banzaicloud/go-cruise-control@latest
+```
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/banzaicloud/go-cruise-control/client"
+	"github.com/banzaicloud/go-cruise-control/api"
+)
+
+func main() {
+    // Initializing Cruise Control client with default configuration
+    cruisecontrol, err := client.NewDefaultClient()
+    if err != nil {
+        panic(err)
+    }
+    
+    // Assembling the request using default values
+    req := api.StateRequestWithDefaults()
+    // Sending the request to the State API
+    resp, err := cruisecontrol.State(req)
+    if err != nil {
+        panic(err)
+    }
+    // Getting the state of the Executor
+    fmt.Println(resp.Result.ExecutorState.State)
+}
+```
+
+## Development
+
+### Prerequisites
+
+* golang `1.17`
+* docker
+* docker-compose
+
+### Development environment
+
+Starting local development environment
+
+```shell
+make env-up
+```
+
+Tearing down the local development environment
+
+```shell
+make env-clean
+```
+
+### Testing
+
+Running integration tests creates a new test environment prior executing the test suite and tears down at the end.
+
+```shell
+make integration-test
+```
+
+Set the `USE_EXISTING=true` environment variable if reuse the existing development/test environment
+for integration testing is the desired behaviour.
+
+```shell
+export USE_EXISTING=true
+make integration-test
+```
+
+## Support
+
+If you encounter problems while using this project than open an issue or talk to us
+on [#kafka-operator](https://banzaicloud.com/invite-slack/) channel on the Banzai Cloud Slack.
```

Comment:
Should we replace this link to `http://community-banzaicloud.slack.com` ?

Reply:
Yep. Agreed.

Reply:
Fixed.